src/Cli/dotnet/commands/dotnet-list/dotnet-list-package/ListPackageReferencesCommandParser.cs

```diff
@@ -61,6 +61,11 @@ internal static class ListPackageReferencesCommandParser
                 ArgumentHelpName = CommonLocalizableStrings.LevelArgumentName
             }.ForwardAsSingle(o => $"--verbosity:{o}");
 
+        public static readonly Option FormatOption = new ForwardedOption<string>("--format", LocalizableStrings.CmdFormatDescription)
+        {
+            ArgumentHelpName = LocalizableStrings.CmdConfig
```

Comment:
This looks like a copy/paste error, I think the ArgumentHelpName should probably be related to this option.

Reply:
Addressed it.